### PR TITLE
webView: Fix text selectable box is visible on long pressing text.

### PR DESCRIPTION
**Pull request by jainkuniya**
screen shot 2018-01-24 at 10 44 17 pm

### CODE REVIEW

File: src/render-html/css.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ html {
   -moz-user-select: none; /* Firefox 2+ */
   -ms-user-select: none; /* IE 10+ */
   user-select: none; /* Standard syntax */
+  -khtml-user-select: none;
```
Comment (borisyankov) [inline src/render-html/css.js]: We should probably do something like the js.js to es3.js conversion, for example, Autoprefixer.
Are you sure khtml is relevant here?
`KDE Konqueror / Apple Safari 1.1 through Safari 2.0, WebKit-Based Browsers`

Comment (borisyankov) [reply src/render-html/css.js]: Maybe UIWebView is so old? This one is relevant I guess: #1716

Comment (jainkuniya) [reply src/render-html/css.js]: hmm
So until we use `WKWebView` to fix this we can take this PR.
I have tested it, it fixes issues. We can include this in beta & definitely improve in future :)

Comment (borisyankov) [reply src/render-html/css.js]: Have a look at the linked PR in React Native and maybe test/comment so they can get more incentive to merge in the main project ;)

**Pull request merged**